pyslurm/pyslurm.pyx

```diff
@@ -2160,7 +2160,7 @@ cdef class job:
             Job_dict[u'start_time'] = self._record.start_time
 
             if self._record.state_desc:
-                Job_dict[u'state_reason'] = self._record.state_desc.replace(" ", "_")
+                Job_dict[u'state_reason'] = self._record.state_desc.decode().replace(" ", "_")
```

Comment:
to be more precise: In a job field, where only a pre-defined set of values defined by slurm are possible. The job name e.g. can of course easily contain non-ASCII characters.